draft-ietf-quic-qpack.md

```diff
@@ -1004,6 +1002,119 @@ Description:
+| 60    | vary                             | accept-encoding                                       |
+| 61    | vary                             | accept-encoding, user-agent                           |
+| 62    | vary                             | origin                                                |
+| 63    | :status                          | 100                                                   |
+| 64    | :status                          | 103                                                   |
+| 65    | :status                          | 204                                                   |
+| 66    | :status                          | 206                                                   |
+| 67    | :status                          | 302                                                   |
+| 68    | :status                          | 400                                                   |
+| 69    | :status                          | 403                                                   |
+| 70    | :status                          | 404                                                   |
+| 71    | :status                          | 421                                                   |
+| 72    | :status                          | 425                                                   |
+| 73    | :status                          | 500                                                   |
+| 74    | :status                          | 503                                                   |
+| 75    | accept-ranges                    | bytes                                                 |
+| 76    | access-control-allow-credentials | FALSE                                                 |
+| 77    | access-control-allow-credentials | TRUE                                                  |
+| 78    | access-control-allow-headers     | *                                                     |
+| 79    | access-control-allow-headers     | cache-control                                         |
+| 80    | access-control-allow-headers     | content-type                                          |
+| 81    | access-control-allow-headers     | x-requested-with                                      |
+| 82    | access-control-allow-methods     | get                                                   |
+| 83    | access-control-allow-methods     | get, post, options                                    |
+| 84    | access-control-allow-methods     | options                                               |
+| 85    | access-control-allow-origin      | *                                                     |
+| 86    | access-control-expose-headers    | content-length                                        |
+| 87    | access-control-request-headers   | content-type                                          |
+| 88    | access-control-request-headers   | x-requested-with                                      |
```

Comment:
People using this header usually don't understand what they're getting; e.g., they use it for access control.

Reply:
I don't remember what it means 5 minutes after I read about it due to lack of hint of intuition, but CORS is used a lot, especially in CDN, so it makes a lot of sense to have, if it it means what I think it means.